pkg/ring/ring_test.go

```diff
@@ -2249,6 +2279,8 @@ func TestShuffleShardWithCaching(t *testing.T) {
 		lcs = append(lcs, lc)
 	}
 
+	time.Sleep(5 * time.Second)
```

Comment:
Trying to make `TestShuffleShardWithCaching` less flaky, not really related to this PR